[2023-11-20T19:19:38.141+0000] {processor.py:154} INFO - Started process (PID=4062) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:19:38.142+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:19:38.142+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:19:38.142+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:19:38.154+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:19:38.144+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 60
    method='GET',
         ^
SyntaxError: invalid syntax
[2023-11-20T19:19:38.155+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:19:38.193+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.056 seconds
[2023-11-20T19:20:08.860+0000] {processor.py:154} INFO - Started process (PID=4126) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:20:08.868+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:20:08.869+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:20:08.869+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:20:08.871+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:20:08.870+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 60
    method='GET',
         ^
SyntaxError: invalid syntax
[2023-11-20T19:20:08.871+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:20:08.925+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.071 seconds
[2023-11-20T19:20:39.150+0000] {processor.py:154} INFO - Started process (PID=4180) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:20:39.151+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:20:39.152+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:20:39.152+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:20:39.160+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:20:39.154+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 60
    method='GET',
         ^
SyntaxError: invalid syntax
[2023-11-20T19:20:39.163+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:20:39.223+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.082 seconds
[2023-11-20T19:21:09.812+0000] {processor.py:154} INFO - Started process (PID=4244) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:21:09.813+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:21:09.814+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:21:09.814+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:21:09.818+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:21:09.817+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 60
    method='GET',
         ^
SyntaxError: invalid syntax
[2023-11-20T19:21:09.818+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:21:09.903+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.098 seconds
[2023-11-20T19:21:40.594+0000] {processor.py:154} INFO - Started process (PID=4299) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:21:40.595+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:21:40.596+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:21:40.595+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:21:40.598+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:21:40.597+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 60
    method='GET',
         ^
SyntaxError: invalid syntax
[2023-11-20T19:21:40.598+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:21:40.627+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.036 seconds
[2023-11-20T19:22:10.931+0000] {processor.py:154} INFO - Started process (PID=4364) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:22:10.932+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:22:10.933+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:22:10.932+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:22:10.935+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:22:10.934+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 60
    method='GET',
         ^
SyntaxError: invalid syntax
[2023-11-20T19:22:10.935+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:22:10.963+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.035 seconds
[2023-11-20T19:22:41.305+0000] {processor.py:154} INFO - Started process (PID=4428) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:22:41.316+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:22:41.317+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:22:41.317+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:22:41.331+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:22:41.318+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 60
    method='GET',
         ^
SyntaxError: invalid syntax
[2023-11-20T19:22:41.332+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:22:41.401+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.106 seconds
[2023-11-20T19:23:11.533+0000] {processor.py:154} INFO - Started process (PID=4481) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:23:11.538+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:23:11.539+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:23:11.538+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:23:11.541+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:23:11.540+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 60
    method='GET',
         ^
SyntaxError: invalid syntax
[2023-11-20T19:23:11.541+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:23:11.571+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.042 seconds
[2023-11-20T19:23:41.789+0000] {processor.py:154} INFO - Started process (PID=4546) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:23:41.791+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:23:41.792+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:23:41.792+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:23:41.795+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:23:41.794+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 60
    method='GET',
         ^
SyntaxError: invalid syntax
[2023-11-20T19:23:41.796+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:23:41.825+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.039 seconds
[2023-11-20T19:24:11.987+0000] {processor.py:154} INFO - Started process (PID=4600) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:24:11.988+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:24:11.989+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:24:11.988+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:24:11.991+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:24:11.990+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 60
    method='GET',
         ^
SyntaxError: invalid syntax
[2023-11-20T19:24:11.991+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:24:12.015+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.030 seconds
[2023-11-20T19:24:42.278+0000] {processor.py:154} INFO - Started process (PID=4664) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:24:42.278+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:24:42.279+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:24:42.279+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:24:42.281+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:24:42.280+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 60
    method='GET',
         ^
SyntaxError: invalid syntax
[2023-11-20T19:24:42.281+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:24:42.303+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.028 seconds
[2023-11-20T19:25:12.375+0000] {processor.py:154} INFO - Started process (PID=4721) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:25:12.376+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:25:12.376+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:25:12.376+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:25:12.378+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:25:12.377+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 60
    method='GET',
         ^
SyntaxError: invalid syntax
[2023-11-20T19:25:12.378+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:25:12.403+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.031 seconds
[2023-11-20T19:25:42.595+0000] {processor.py:154} INFO - Started process (PID=4787) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:25:42.595+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:25:42.596+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:25:42.596+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:25:42.597+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:25:42.597+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 60
    method='GET',
         ^
SyntaxError: invalid syntax
[2023-11-20T19:25:42.598+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:25:42.623+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.031 seconds
[2023-11-20T19:26:13.192+0000] {processor.py:154} INFO - Started process (PID=4851) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:26:13.193+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:26:13.193+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:26:13.193+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:26:13.196+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:26:13.195+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 60
    method='GET',
         ^
SyntaxError: invalid syntax
[2023-11-20T19:26:13.196+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:26:13.231+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.042 seconds
[2023-11-20T19:26:44.153+0000] {processor.py:154} INFO - Started process (PID=4913) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:26:44.155+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:26:44.156+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:26:44.156+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:26:44.158+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:26:44.157+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 60
    method='GET',
         ^
SyntaxError: invalid syntax
[2023-11-20T19:26:44.158+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:26:44.199+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.051 seconds
[2023-11-20T19:27:15.193+0000] {processor.py:154} INFO - Started process (PID=4971) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:27:15.194+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:27:15.203+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:27:15.195+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:27:15.205+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:27:15.204+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 60
    method='GET',
         ^
SyntaxError: invalid syntax
[2023-11-20T19:27:15.206+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:27:15.289+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.099 seconds
[2023-11-20T19:27:46.020+0000] {processor.py:154} INFO - Started process (PID=5035) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:27:46.032+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:27:46.032+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:27:46.032+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:27:46.035+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:27:46.034+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 60
    method='GET',
         ^
SyntaxError: invalid syntax
[2023-11-20T19:27:46.035+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:27:46.092+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.074 seconds
[2023-11-20T19:28:16.555+0000] {processor.py:154} INFO - Started process (PID=5100) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:28:16.565+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:28:16.568+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:28:16.567+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:28:16.574+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:28:16.573+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 60
    method='GET',
         ^
SyntaxError: invalid syntax
[2023-11-20T19:28:16.595+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:28:16.679+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.127 seconds
[2023-11-20T19:28:47.511+0000] {processor.py:154} INFO - Started process (PID=5156) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:28:47.518+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:28:47.519+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:28:47.519+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:28:47.522+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:28:47.521+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 60
    method='GET',
         ^
SyntaxError: invalid syntax
[2023-11-20T19:28:47.524+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:28:47.566+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.131 seconds
[2023-11-20T19:29:17.759+0000] {processor.py:154} INFO - Started process (PID=5222) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:29:17.760+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:29:17.761+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:29:17.760+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:29:17.764+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:29:17.762+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 60
    method='GET',
         ^
SyntaxError: invalid syntax
[2023-11-20T19:29:17.764+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:29:17.800+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.044 seconds
[2023-11-20T19:29:48.021+0000] {processor.py:154} INFO - Started process (PID=5278) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:29:48.023+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:29:48.024+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:29:48.023+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:29:48.026+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:29:48.025+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 60
    method='GET',
         ^
SyntaxError: invalid syntax
[2023-11-20T19:29:48.029+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:29:48.076+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.064 seconds
[2023-11-20T19:30:18.255+0000] {processor.py:154} INFO - Started process (PID=5342) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:30:18.257+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:30:18.258+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:30:18.258+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:30:18.260+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:30:18.259+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 60
    method='GET',
         ^
SyntaxError: invalid syntax
[2023-11-20T19:30:18.261+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:30:18.315+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.063 seconds
[2023-11-20T19:30:48.735+0000] {processor.py:154} INFO - Started process (PID=5397) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:30:48.743+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:30:48.744+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:30:48.744+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:30:48.746+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:30:48.745+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 60
    method='GET',
         ^
SyntaxError: invalid syntax
[2023-11-20T19:30:48.746+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:30:48.788+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.060 seconds
[2023-11-20T19:31:19.611+0000] {processor.py:154} INFO - Started process (PID=5461) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:31:19.611+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:31:19.612+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:31:19.612+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:31:19.614+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:31:19.613+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 60
    method='GET',
         ^
SyntaxError: invalid syntax
[2023-11-20T19:31:19.615+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:31:19.671+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.063 seconds
[2023-11-20T19:31:50.192+0000] {processor.py:154} INFO - Started process (PID=5517) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:31:50.193+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:31:50.193+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:31:50.193+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:31:50.196+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:31:50.195+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 60
    method='GET',
         ^
SyntaxError: invalid syntax
[2023-11-20T19:31:50.196+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:31:50.227+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.038 seconds
[2023-11-20T19:32:01.141+0000] {processor.py:154} INFO - Started process (PID=5545) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:32:01.141+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:32:01.142+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:32:01.142+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:32:04.169+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:32:04.915+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:32:04.915+0000] {manager.py:507} INFO - Created Permission View: can delete on DAG:user_processing
[2023-11-20T19:32:04.931+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:32:04.931+0000] {manager.py:507} INFO - Created Permission View: can edit on DAG:user_processing
[2023-11-20T19:32:04.947+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:32:04.947+0000] {manager.py:507} INFO - Created Permission View: can read on DAG:user_processing
[2023-11-20T19:32:04.948+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:32:04.947+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-11-20T19:32:05.161+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:32:05.160+0000] {dag.py:2606} INFO - Creating ORM DAG for user_processing
[2023-11-20T19:32:05.175+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:32:05.175+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-11-20T00:00:00+00:00, run_after=2023-11-21T00:00:00+00:00
[2023-11-20T19:32:05.229+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 4.090 seconds
[2023-11-20T19:32:35.284+0000] {processor.py:154} INFO - Started process (PID=5610) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:32:35.285+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:32:35.285+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:32:35.285+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:32:35.561+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:32:35.777+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:32:35.776+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-11-20T19:32:35.800+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:32:35.800+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-11-20T00:00:00+00:00, run_after=2023-11-21T00:00:00+00:00
[2023-11-20T19:32:35.829+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.548 seconds
[2023-11-20T19:33:05.953+0000] {processor.py:154} INFO - Started process (PID=5675) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:33:05.954+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:33:05.954+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:33:05.954+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:33:06.327+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:33:06.783+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:33:06.783+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-11-20T19:33:06.890+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:33:06.890+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-11-20T00:00:00+00:00, run_after=2023-11-21T00:00:00+00:00
[2023-11-20T19:33:06.944+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.993 seconds
[2023-11-20T19:33:37.734+0000] {processor.py:154} INFO - Started process (PID=5731) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:33:37.736+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:33:37.737+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:33:37.737+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:33:38.135+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:33:38.317+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:33:38.317+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-11-20T19:33:38.338+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:33:38.338+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-11-20T00:00:00+00:00, run_after=2023-11-21T00:00:00+00:00
[2023-11-20T19:33:38.359+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.635 seconds
[2023-11-20T19:34:08.719+0000] {processor.py:154} INFO - Started process (PID=5796) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:34:08.720+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:34:08.721+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:34:08.721+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:34:09.155+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:34:09.349+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:34:09.349+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-11-20T19:34:09.371+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:34:09.371+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-11-20T00:00:00+00:00, run_after=2023-11-21T00:00:00+00:00
[2023-11-20T19:34:09.390+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.675 seconds
[2023-11-20T19:34:39.996+0000] {processor.py:154} INFO - Started process (PID=5853) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:34:39.997+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:34:39.997+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:34:39.997+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:34:40.360+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:34:40.624+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:34:40.623+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-11-20T19:34:40.660+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:34:40.659+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-11-20T00:00:00+00:00, run_after=2023-11-21T00:00:00+00:00
[2023-11-20T19:34:40.684+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.692 seconds
[2023-11-20T19:35:10.851+0000] {processor.py:154} INFO - Started process (PID=5919) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:35:10.852+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:35:10.853+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:35:10.853+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:35:11.155+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:35:11.309+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:35:11.308+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-11-20T19:35:11.330+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:35:11.330+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-11-20T00:00:00+00:00, run_after=2023-11-21T00:00:00+00:00
[2023-11-20T19:35:11.352+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.504 seconds
[2023-11-20T19:35:41.447+0000] {processor.py:154} INFO - Started process (PID=5976) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:35:41.478+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:35:41.478+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:35:41.478+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:35:43.415+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:35:44.420+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:35:44.419+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-11-20T19:35:44.533+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:35:44.533+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-11-20T00:00:00+00:00, run_after=2023-11-21T00:00:00+00:00
[2023-11-20T19:35:44.655+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 3.224 seconds
[2023-11-20T19:36:15.161+0000] {processor.py:154} INFO - Started process (PID=6040) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:36:15.162+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:36:15.175+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:36:15.163+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:36:15.986+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:36:16.544+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:36:16.543+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-11-20T19:36:16.616+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:36:16.616+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-11-20T00:00:00+00:00, run_after=2023-11-21T00:00:00+00:00
[2023-11-20T19:36:16.659+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 1.514 seconds
[2023-11-20T19:36:47.392+0000] {processor.py:154} INFO - Started process (PID=6095) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:36:47.393+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:36:47.393+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:36:47.393+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:36:48.108+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:36:48.570+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:36:48.570+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-11-20T19:36:48.603+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:36:48.603+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-11-20T00:00:00+00:00, run_after=2023-11-21T00:00:00+00:00
[2023-11-20T19:36:48.640+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 1.252 seconds
[2023-11-20T19:37:18.966+0000] {processor.py:154} INFO - Started process (PID=6159) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:37:18.966+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:37:18.967+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:37:18.967+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:37:19.241+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:37:19.398+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:37:19.398+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-11-20T19:37:19.419+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:37:19.419+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-11-20T00:00:00+00:00, run_after=2023-11-21T00:00:00+00:00
[2023-11-20T19:37:19.440+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.477 seconds
[2023-11-20T19:37:49.498+0000] {processor.py:154} INFO - Started process (PID=6224) to work on /opt/airflow/dags/user_processing.py
[2023-11-20T19:37:49.498+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2023-11-20T19:37:49.499+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:37:49.499+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2023-11-20T19:37:49.750+0000] {processor.py:766} INFO - DAG(s) dict_keys(['user_processing']) retrieved from /opt/airflow/dags/user_processing.py
[2023-11-20T19:37:49.921+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:37:49.921+0000] {dag.py:2585} INFO - Sync 1 DAGs
[2023-11-20T19:37:49.941+0000] {logging_mixin.py:120} INFO - [2023-11-20T19:37:49.941+0000] {dag.py:3340} INFO - Setting next_dagrun for user_processing to 2023-11-20T00:00:00+00:00, run_after=2023-11-21T00:00:00+00:00
[2023-11-20T19:37:49.961+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/user_processing.py took 0.466 seconds
